src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -190,4 +201,43 @@ private CommandResult executeCommand(String commandText) throws CommandException
             throw e;
         }
     }
+
+    public void setOnButtonPressedListener() {
```

Comment:
I think it would be better to abstract this code to an AutoCompletePanelManager or something. Either way I don't think the body of the code should be in the MainWindow class.